### PR TITLE
CollapsibleCard to MUI Accordion

### PR DESCRIPTION
**Pull request by d88naimi**
Clip to show functionality and styles 
https://www.screencast.com/t/xenN5Svbw

### CODE REVIEW

File: src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/Unpublish/Unpublish.js [hunk 1]
```diff
@@ -32,22 +29,23 @@ export const Unpublish = memo(function Unpublish(props) {
...
-      header={
-        <Fragment>
-          <FontAwesomeIcon icon={faUnlink} />
-          &nbsp;Unpublish
-        </Fragment>
-      }
-    >
-      <CardContent>
-        <p>
+    <Accordion sx={{ m: "16px !important" }}>
```
Comment (agalin920) [inline src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/Unpublish/Unpublish.js:32]: Should use system and !important shouldn't be required

File: src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/Unpublish/Unpublish.js [hunk 2]
```diff
@@ -32,22 +29,23 @@ export const Unpublish = memo(function Unpublish(props) {
...
+      <AccordionSummary
+        expandIcon={<ExpandMoreIcon />}
+        aria-controls="panel1a-content"
```
Comment (agalin920) [inline src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/Unpublish/Unpublish.js:35]: these aria and ids are for demo purposes

File: src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/Unpublish/Unpublish.js [hunk 3]
```diff
@@ -32,22 +29,23 @@ export const Unpublish = memo(function Unpublish(props) {
...
+          <UnpublishedIcon fontSize="small" /> Unpublish
+        </Typography>
+      </AccordionSummary>
+      <AccordionDetails>
+        <Typography>
           By unpublishing this content it will no longer be served if the URL is
           requested. The URL will return a 404 not found response.
-        </p>
-      </CardContent>
-      <CardFooter>
```
Comment (agalin920) [inline src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/Unpublish/Unpublish.js:50]: Footer buttons seem more fitting to be wrapped within <AccordionActions>

Comment (d88naimi) [reply src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/Unpublish/Unpublish.js:50]: @agalin920 Wrapped within Typography?

File: src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/Unpublish/Unpublish.js [hunk 4]
```diff
@@ -32,34 +30,36 @@ export const Unpublish = memo(function Unpublish(props) {
...
+            {" "}
+            <UnpublishedIcon fontSize="small" /> Unpublish
+          </Typography>
+        </AccordionSummary>
+        <AccordionDetails
+          sx={{ display: "flex", flexDirection: "column", gap: "16px" }}
         >
-          Unpublish
-        </Button>
-      </CardFooter>
```
Comment (agalin920) [inline src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/Unpublish/Unpublish.js:62]: CardFooter maps to AccordionActions. Seeing it missing a lot on the PR.

**Review by agalin920 — CHANGES_REQUESTED** [review]

**Review by agalin920 — APPROVED** [review]
Approved

**Pull request merged**